network/router.go

```diff
@@ -460,7 +460,9 @@ func (r *Router) receiveServerIdentity(c Conn) (*ServerIdentity, error) {
 			log.Lvl4(r.address, "Public key from CommonName and ServerIdentity match:", pub)
 		} else {
 			// We get here for TCPConn && !tls.Conn. Make them wish they were using TLS...
-			if !r.UnauthOk {
+			// Don't show error in simulations.
+			if !r.UnauthOk &&
```

Comment:
Please arrange that UnauthOk is set to true by the simulation system. Add "c.Router.UnauthOk = true" in runsimul.go, right before c.Start()

Disabling security warnings via substring matches is not ok. :)

Reply:
OK - now I understand ;) sounds good...